lib/Activity/Provider/Base.php

```diff
@@ -126,7 +126,7 @@ protected function getRoom(IL10N $l, Room $room) {
 		return [
 			'type' => 'call',
 			'id' => $room->getId(),
-			'name' => $room->getName() ?: $l->t('a call'),
```

Comment:
This should probably remain "a call" since it's of type call?

Reply:
Actually might be right, since notifier users different stuff for sending a call.